ansible/bootstrap.yml

```diff
@@ -107,6 +107,32 @@
       delay: 1
       when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
 
+    - name: Verify Consul ACL deny policy - list services without token, does not show servie consul
+      uri:
+        url: http://127.0.0.1:8500/v1/catalog/services
+        status_code: 200
+        body_format: json
+      register: services_payload_1
+      until: services_payload_1.status == 200
+      retries: 5
+      delay: 1
+      failed_when: services_payload_1.json["consul"] is defined
+      when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+
+    - name: Verify Consul ACL deny policy - list services, with correct token, contains service consul
+      uri:
+        url: http://127.0.0.1:8500/v1/catalog/services
+        status_code: 200
+        headers:
+          X-Consul-Token: "{{ token }}"
+        body_format: json
+      register: services_payload_2
+      until: services_payload_2.status == 200
+      retries: 5
+      delay: 1
+      when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+      failed_when: services_payload_2.json["consul"] is not defined
```

Comment:
I feel like we are getting to a point where these tests could be separated into its own playbook. WDYT?

Reply:
Yeah. I agree. Might be useful to have a separate folder under ansible

Reply:
Yeah, definitely add a folder, because there's a lot of playbooks under `ansible/ ` right now.

Reply:
If you feel like it we could just make an issue for this, and merge this PR as is first!

Reply:
done it here 3b8aa51